run-tests.sh

```diff
@@ -483,21 +483,22 @@ if [ -z "$checkForPresenceOfMetadataIngestionSowerJob" ]; then
 fi
 
 # # Study Viewer test
-runStudyViewerTests=false
-#run for data-portal/requestor/gen3-qa/gitops-qa/cdis-manifest repo
-if [[ ! ("$service" =~ ^(data-portal|requestor|gen3-qa)$ || $testedEnv == *"niaid"*) ]]; then
-  echo "Disabling study-viewer test"
-  donot "@studyViewer"
-else
-  if [[ $(curl -s "$portalConfigURL" | jq 'contains({studyViewerConfig})') == "true" ]]; then
-    if (g3kubectl get pods --no-headers -l app=requestor | grep requestor) > /dev/null 2>&1; then
-      echo "### Study-Viewer is deployed"
-      runStudyViewerTests=true
-    fi
-  fi
-fi
+# runStudyViewerTests=false
```

Comment:
let's not just comment it out and forget about it - let's create a ticket to fix this test, and add the ticket number here

Reply:
https://ctds-planx.atlassian.net/browse/GQE-211